src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -293,4 +314,21 @@ public boolean getHasError() {
     public void setHasError(boolean value) {
         this.hasError = value;
     }
+    
+    public String getCreatorEmail() {
+        return creatorEmail;
+    }
+    
+    public void setCreatorEmail(String creatorEmail) {
+        this.creatorEmail = creatorEmail;
+    }
+    
+    public String getStudentsDataAsString() {
+        JSONObject json = new JSONObject(studentsData);
+        return Sanitizer.sanitizeForHtml(json.toString());
```

Comment:
Are we converting the `studentData` to JSON so that it can be easily converted to string? If so, maybe leave a comments so future readers will not be confused by the JSON conversion.